src/axom/mir/MeshTester.hpp

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) 2017-2025, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*!
+ * \file MeshTester.hpp
+ * 
+ * \brief Contains the specification for the MeshTester class.
+ * 
+ */
```

Comment:
Can we create a ticket to reimplement some of these test cases using Klee in a follow-up PR?
The ones we can't reproduce w/ Klee, (e.g. b/c they give explicit values) should probably be moved to `axom_data`

Reply:
That would introduce **new** klee/quest dependencies into MIR. Is that worth it?

Reply:
clearly, this would only be for the examples that create volume fraction data from geometry, as opposed to the ones that have explicit volume fractions